src/components/Selector/Selector.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bulma-components';
+
+export class Selector extends React.Component {
+  state = {
+    selectedGoods: [],
+  };
+
+  addItem = (item) => {
+    this.setState(prevState => ({
+      selectedGoods: [...prevState.selectedGoods, item],
+    }));
+  };
```

Comment:
Your function still adds multiple same items to the list. You can prevent this by adding check here like
`if(!this.state.selectedGoods.includes(item)) {...}`